plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,189 @@
+from proxmoxer import ProxmoxAPI
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.general.plugins.module_utils.proxmox import (
+    proxmox_auth_argument_spec, ProxmoxAnsible, proxmox_to_ansible_bool)
+
+import urllib3
+
+urllib3.disable_warnings()
+
+DOCUMENTATION = '''
+---
+module: proxmox_backup_info
+short_description: Retrieve information about one or more Proxmox VE backups
+version_added: 2.2.0
```

Comment:
```suggestion
version_added: 9.0.0
```

Reply:
Fix in commit [49d6df9541b7e86c582d5b342f3d1677f4523f5b](https://github.com/ansible-collections/community.general/pull/8301/commits/49d6df9541b7e86c582d5b342f3d1677f4523f5b)